README.md

```diff
@@ -5,6 +5,7 @@ Published destination: https://commonprofiles.care/
 More info
 * https://github.com/FHIR/auto-ig-builder/blob/master/README.md
 * https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build
+* https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build/search/svenskaindustriprofiler.2Ffhir
```

Comment:
It is very good that we redirect to the specific page.
Now we need a link to https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build?
Can't we remove line 7 now?

Reply:
> It is very good that we redirect to the specific page. Now we need a link to https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build? Can't we remove line 7 now?

Fixed!